packages/assets-controllers/src/assetsUtil.ts

```diff
@@ -292,3 +293,68 @@ export async function reduceInBatchesSerially<
   const finalResult = workingResult as Result;
   return finalResult;
 }
+
+/**
+ * Maps an OpenSea V2 NFT to the V1 schema.
+ * Should work for both 'get_nft' and 'list_nfts_by_account' responses.
+ * @param nft - The V2 NFT to map.
+ * @returns The NFT in the V1 schema.
+ */
+export function mapOpenSeaNftV2ToV1(nft: any): ApiNft {
+  return {
+    token_id: nft.identifier,
+    num_sales: null,
+    background_color: null,
+    image_url: nft.image_url,
+    image_preview_url: nft.image_url,
+    image_thumbnail_url: nft.image_url,
+    image_original_url: null,
```

Comment:
TODO: In v2 I only get a `image_url` instead of all these variants.  Determine whether `image_url` should stand-in for some of them, or if the clients can handle falling back to image_url